### PR TITLE
Implementation of T5 training

### PR DESCRIPTION
**Pull request by orzylowski**

### CODE REVIEW

File: polish_labels.json [hunk 1]
```diff
@@ -0,0 +1,62 @@
+{
+    "alarm_query": "alarm_zapytanie",
+    "alarm_remove": "alarm_usuwanie",
+    "alarm_set": "alarm_ustawianie",
+    "audio_volume_down": "dźwięk_ciszej",
```
Comment (orzylowski) [inline polish_labels.json:5]: Maybe I should leave `'audio'` or change it to `'głośność'` (but it is even longer than `'dźwięk'` and has more Polish characters)?

File: polish_labels.json [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    "cooking_recipe": "gotowanie_przepis",
+    "datetime_convert": "data_czas_konwersja",
+    "datetime_query": "data_czas_zapytanie",
+    "email_addcontact": "email_dodawanie_kontaktu",
+    "email_query": "email_zapytanie",
+    "email_querycontact": "email_zapytanie_kontakt",
+    "email_sendemail": "email_wysyłanie",
+    "general_greet": "ogólne_powitanie",
+    "general_joke": "ogólne_żart",
+    "general_quirky": "ogólne_zapytanie",
```
Comment (orzylowski) [inline polish_labels.json:22]: `'query'` seems to fit here better than `'quirky'` - that's way I use `'zapytanie'` here

File: polish_labels.json (hunk 1 above)
Comment (Tsilkow) [reply polish_labels.json:5]: `'dźwięk'` is definetely the best Polish translation, `'audio'` functions in Polish language so keeping it is acceptable as well.

I think we should keep `'dźwięk'`.

File: polish_labels.json [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    "iot_hue_lightoff": "urządzenie_światło_wyłączenie",
+    "iot_hue_lighton": "urządzenie_światło_włączenie",
+    "iot_hue_lightup": "urządzenie_światło_rozjaśnienie",
+    "iot_wemo_off": "urządzenie_gniazdko_wyłącz",
+    "iot_wemo_on": "urządzenie_gniazdko_włącz",
+    "lists_createoradd": "listy_tworzenie_lub_dodawanie",
+    "lists_query": "listy_zapytanie",
+    "lists_remove": "listy_usuwanie",
+    "music_dislikeness": "muzyka_lubienie",
+    "music_likeness": "muzyka_nielubienie",
```
Comment (Tsilkow) [inline polish_labels.json]: Translations of `"music_dislikeness"` and `"music_likeness"` have their meaning switched.

**Review by Tsilkow — CHANGES_REQUESTED** [review]

**Review by Tsilkow — APPROVED** [review]

**Pull request merged**